src/VImageCleaning.cpp

```diff
@@ -424,6 +424,99 @@ bool VImageCleaning::InitNNImageCleaning()
 	return true;
 }
 
+/*
+ * Calculate averaged IPR graph over all telescopes when there is not enough statistics
+ * Call it if flag fIPRAverageTel > 0.
+ */
+TGraphErrors* VImageCleaning::calculateIPRGraphsAveragedAllTelescopes( unsigned int iSummationWindow )
+{
+
+	cout << "VImageCleaning::calculateIPRGraphsAveragedAllTelescopes: doing average of IPR graphs." << endl;
+	TGraphErrors* IPRGraphAverageTel = new TGraphErrors();
+	
+	char InitTelID = fData->getTelID();
+        int NTelAverage = fData->getNTel();
```

Comment:
Should have been more specific: getNTel() returns you always 4 for VERITAS, even for 3-telescope runs.

The loop should like like:
```
for( unsigned int i = 0; i < fData->getTeltoAna().size(); i++ )
{
     fData->setTelID( fData->getTeltoAna()[i] );
     ...
    ....
}
```
The number of available telescopes is `fData->getTeltoAna().size()`